src/main/java/org/apache/maven/plugins/help/SystemMojo.java

```diff
@@ -67,14 +67,10 @@ public void execute() throws MojoExecutionException {
         message.append(StringUtils.repeat("=", LINE_LENGTH)).append(LS);
         message.append("Environment Variables").append(LS);
         message.append(StringUtils.repeat("=", LINE_LENGTH)).append(LS);
-        try {
-            Properties envVars = CommandLineUtils.getSystemEnvVars();
-            for (String key : envVars.stringPropertyNames()) {
-                message.append(LS);
-                message.append(key).append("=").append(envVars.getProperty(key));
-            }
-        } catch (IOException e) {
-            getLog().warn("Unable to get the environment variables: " + e.getMessage());
+        Properties envVars = CommandLineUtils.getSystemEnvVars();
```

Comment:
Why not `System.getenv()`

Reply:
logic is more advanced https://github.com/codehaus-plexus/plexus-utils/blob/6714fe016acb5a8a503241c5f2b86438f387ac9f/src/main/java/org/codehaus/plexus/util/cli/CommandLineUtils.java#L374

Reply:
It is only used for display - so maybe more complicated logic is not needed

Reply:
Up to you